os/arch/arm/src/amebad/Make.defs

```diff
@@ -199,3 +199,17 @@ else
 EXTRA_LIBS += ${TOPDIR}/board/rtl8721csm/src/libs/cmse_implib.a
 endif
 endif
+

```

Comment:
Can you confirm library inclusion in commit message

Reply:
Done. Thanks.